charts/mod-consortia-keycloak/values.yaml

```diff
@@ -70,9 +70,9 @@ javaOpts:
 extraJavaOpts: []
   # - "-Dlog_level=DEBUG"
 
-extraEnvVars: []
-  # - name: FOO
-  #   value: bar
+extraEnvVars: 
+    - name: OKAPI_URL
```

Comment:
I missed mentioning this in the previous review. Please consider using a two-space indent instead of tabulation, including in the other files.

Reply:
@epam-avramenko
Fixed on your request.